project_report/templates/report_base.html

```diff
@@ -4,72 +4,152 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" crossorigin="anonymous">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" crossorigin="anonymous">
     <style type="text/css">
-        #edgen_logo { width: 150px; }
-        h1 { font-size: 2.7em; }
-        h2 { font-size: 2.0em; }
-        h3 { font-size: 1.3em; }
-        .pagebreak { page-break-before: always; }
+        #edgen_logo { width: 70px; margin-left: auto; margin-right: auto; display: inline-block;; }
+        h2 { font-size: 1.7em; line-height: 160%; font-family:Avantgarde; sans-serif; }
+        h4 { font-size: 2.7em; line-height: 300%; font-family:Avantgarde; sans-serif; }
+
+        p.font1 {
+            font-size: 1.4em;
+            line-height: 140%;
+            font-family: Avantgarde;
+            sans-serif;
```

Comment:
I'm not sure if this is valid CSS. `sans-serif` is a font style to be used in `font-family` in a comma-separated list. The browser then goes over the list of styles and falls back on the next one if a font style is unavailable. This also means that in h2 and h4 above:

    font-family: Avantgarde; sans-serif;

should be:

    font-family: Avantgarde, sans-serif;